axelor-crm/src/main/resources/views/Opportunity.xml

```diff
@@ -261,6 +265,14 @@
     	<field name="description" expr="eval: _partner.description"/>
     </action-record>
     
+    <action-attrs name="action-opportunity-attrs-description-panel-show">
```

Comment:
use 'If' condition instead of 'action-attrs' like used for all other apps setting check.